plugins/mql-language/src/test/java/org/polypheny/db/mql/mql2alg/dql/Mql2AlgFindTest.java

```diff
@@ -38,11 +37,12 @@
 import org.polypheny.db.mql.mql2alg.Mql2AlgTest;
 import org.polypheny.db.nodes.Operator;
 import org.polypheny.db.rex.RexCall;
-import org.polypheny.db.rex.RexInputRef;
+import org.polypheny.db.rex.RexIndexRef;
 import org.polypheny.db.rex.RexLiteral;
 import org.polypheny.db.rex.RexNode;
+import org.polypheny.db.type.entity.PolyValue;
 
-@Ignore // this test is an extreme time sink, use integration tests
+@Disabled // this test is an extreme time sink, use integration tests
```

Comment:
Then we get rid of it and migrate tests to integration tests.

Reply:
Removed.